Dockerfile

```diff
@@ -22,7 +24,7 @@ RUN . $NVM_DIR/nvm.sh \
     && nvm alias default $NODE_VERSION \
     && nvm use default
 
-RUN npm install -g yarn
+RUN npm install -g yarn@1.19.1
```

Comment:
What happens as our local machines' yarn versions get out of sync with this one?

Reply:
I think we issue a PR to update the Dockerfile when our versions change. 

Motivation comes from 12 factor app (https://12factor.net/dependencies) and we should explicitly mark our versions in the docker file so that stale versions don't get cached, or updated out from under us.